builder/vsphere/common/output_config.go

```diff
@@ -21,12 +21,22 @@ type OutputConfig struct {
 	// created, must be empty prior to running the builder. By default this is
 	// "output-BUILDNAME" where "BUILDNAME" is the name of the build.
 	OutputDir string `mapstructure:"output_directory" required:"false"`
+	// The permission of the directories newly created to store artifacts,
+	// possibly including the directory speified by "output_directory" and its
+	// ancestor directories. By default this is "0750". You should express the
```

Comment:
I think you mean "child" directories here?

Reply:
I see what you mean. You mean all the directories that have to be created via mkdirall. There's probably a more explicit way to phrase this -- how about "The permissions to apply to the the "output_directory", and to any parent directories that get created for output_directory."

Reply:
Thank you for your reply. 

> The permissions to apply to the the "output_directory", and to any parent directories that get created for output_directory.

That's exactly what I mean. I'll fix it now.

Reply:
https://github.com/hashicorp/packer/pull/9704/commits/a53e81d9a276be1fdb69266b8b31e19728a6d346